tests/fourier/test_coefficients.py

```diff
@@ -258,7 +258,7 @@ def test_anti_aliasing(self, circuit, inpt, degree):
         """Test that the coefficients obtained through anti-aliasing are the
         same as the ones when we don't anti-alias at the correct degree."""
         coeffs_regular = coefficients(circuit, len(inpt), degree, lowpass_filter=False)
-        coeffs_anti_aliased = coefficients(circuit, len(inpt), degree)
+        coeffs_anti_aliased = coefficients(circuit, len(inpt), degree, lowpass_filter=True)
```

Comment:
What does `lowpass_filter` do? :thinking:

Reply:
```
lowpass_filter (bool) – If True, a simple low-pass filter is applied prior to computing the set of coefficients in order to filter out frequencies above the given degree. See examples below.
```
It defaults to `False`.  So `coeffs_anti_aliased` were not in fact anti-aliased, but regular.